src/query_ctx.c

```diff
@@ -274,7 +273,8 @@ void QueryCtx_Rollback(void) {
 
 	if(ctx->undo_log == NULL) return;
 	
-	UndoLog_Rollback(&ctx->undo_log);
```

Comment:
is it possible to free and NULL set the `undo_log` within `UndoLog_Roolback` function ?